frontend/src/context/config/configSlice.tsx

```diff
@@ -11,14 +11,15 @@ import {
   initialSourceControlState,
 } from '@src/context/config/sourceControl/sourceControlSlice'
 import { REQUIRED_DATA } from '@src/constants'
+import dayjs from 'dayjs'
 export interface BasicConfigState {
   isProjectCreated: boolean
   basic: {
     projectName: string
     calendarType: string
     dateRange: {
-      startDate: string | null
-      endDate: string | null
+      startDate: string | number | null
```

Comment:
Can we keep it simple,just use string and null

Reply:
I have no idea about that , cause I met some problem by doing this.
I need to change the type of date for verify, but it will cause type mismatch.

Reply:
fix that!